`ConnectionHandle` not found when using `axum` but not `tungstenite` feature

```
   Compiling samod v0.9.0
error[E0425]: cannot find type `ConnectionHandle` in this scope
   --> /home/kaspar/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/samod-0.9.0/src/websocket.rs:272:17
    |
272 |     ) -> Result<ConnectionHandle, crate::Stopped> {
    |                 ^^^^^^^^^^^^^^^^ not found in this scope
    |
help: consider importing this struct through its public re-export
    |
  1 + use crate::ConnectionHandle;
    |
```

Adding `tungstenite` to features works around this. Previously we had only `features = ["tokio", "axum"]`

--- a/samod/src/websocket.rs
+++ b/samod/src/websocket.rs
@@ -3,7 +3,7 @@ use futures::TryStreamExt;
 use futures::{Sink, SinkExt, Stream, StreamExt};
 
 use crate::Repo;
-#[cfg(feature = "tungstenite")]
+#[cfg(any(feature = "tungstenite", feature = "axum"))]
 use crate::connection::ConnectionHandle;
 
 #[cfg(feature = "tungstenite")]

